apps/core/src/components/blocksuite/block-suite-header-title/operation-menu.tsx

```diff
@@ -78,9 +80,11 @@ export const PageMenu = ({ rename, pageId }: PageMenuProps) => {
   }, [mode, setSetting, t]);
   const handleOnConfirm = useCallback(() => {
     removeToTrash(pageId);
+    jumpToSubPath(workspace.id, WorkspaceSubPath.ALL);
+    //toast does't work
     toast(t['com.affine.toastMessage.movedTrash']());
```

Comment:
Can you also change this toast to using `pushNotification`? You can search for it for its usage

Reply:
> Can you also change this toast to using `pushNotification`? You can search for it for its usage

Done replaced `toast `to `pushNotification` and added `moveToTrash` message description that was suggested in #4282  to all language packs